burger/toppings/packetinstructions.py

```diff
@@ -392,7 +392,7 @@ def operations(classloader, cf, classes, verbose, method, arg_names, special_fie
                     stack.insert(
                         -3 if stack[-3].category == 2 else -4, stack[-1]
                     )
-            elif mnemonic == "return":
+            elif mnemonic in ("return", "lreturn", "freturn", "dreturn", "areturn"):
```

Comment:
This seems to be missing `ireturn`, but otherwise seems reasonable.

Reply:
Oh good catch, fixed :)